prometheus_client/exposition.py

```diff
@@ -122,7 +122,7 @@ def prometheus_app(environ, start_response):
             # Bake output
             status, header, output = _bake_output(registry, accept_header, params)
         # Return output
-        start_response(status, [header])
+        start_response(str(status), [header])
```

Comment:
I wonder if lines 118 and 119 should just be cast as `str` similar to what the end of `_bake_output` does? Do you only see this issue for requests going to `/favicon.ico` or other locations as well?